src/Notes.jsx

```diff
@@ -149,6 +158,58 @@ export default class Notes extends Component {
 
 	}
 
+	onEditNote(note) {
+
+		const notes = this.state.notes;
+		const noteIndex = this.state.notes.indexOf(note);
+
+		if (noteIndex === -1) {
+			return;
+		}
+
+		// switch edit mode
+		notes[noteIndex].editing = true;
+
+		// todo set autofocus on textarea (like textareaRef)
```

Comment:
Still need to implement this. Doing this will likely make the styling of the edit box match the "addNote" text box.